ui-admin/src/user/DeleteUserModal.test.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import { mockPortal } from 'test-utils/mocking-utils'
+import  { setupRouterTest } from 'test-utils/router-testing-utils'
+import { render, screen } from '@testing-library/react'
+import DeleteUserModal from './DeleteUserModal'
+import userEvent from '@testing-library/user-event'
+import { mockAdminUser } from '../test-utils/user-mocking-utils'
+
+describe('DeleteUserModal', () => {
+  test('enables Remove button when user completes matching confirmation string', async () => {
+    //Arrange
+    const user = userEvent.setup()
+    const portal = mockPortal()
+    const subjUser = mockAdminUser(false)
+    subjUser.username = 'user@email'
+
+    const { RoutedComponent } = setupRouterTest(<DeleteUserModal
+      subjUser={subjUser}
+      portal={portal}
+      userDeleted={jest.fn()}
+      onDismiss={jest.fn()}/>)
+    render(RoutedComponent)
+
+    //Assert page content
+    const confirmRemoveUserInput = screen.getByText('Confirm by typing "remove user@email" below')
+    const removeButton = screen.getByText('Remove user')
+    expect(removeButton).toBeDisabled()
+
+    //Act
+    await user.type(confirmRemoveUserInput, 'remove user@email')
+
+    //Assert action
```

Comment:
Ha! I was trying to just follow the pattern I saw elsewhere. Apparently that's not our standard. :-) 
Will change.

Reply:
Yeah, Brian liked those, and I gave him the benefit of a doubt because he was very enthusiastic about them.  But if you are not, let's get rid of them :)